src/aircraft_cmd.cpp

```diff
@@ -1693,6 +1693,16 @@ static void AircraftEventHandler_Flying(Aircraft *v, const AirportFTAClass *apc)
 			current = current->next.get();
 		}
 	}
+
+	/**
+	 * If the aircraft is in the air and there are no valid airport left then crash.
+	 */
+	if(st == nullptr || st->airport.tile == INVALID_TILE) {
+		if(v->current_order_time > 15000){
```

Comment:
Missing spaces. But more importantly, where did you get 15000 from? Why pick this number?

```suggestion
		if (v->current_order_time > 15000) {
```

Reply:
15000 is somewhat 2.5 minutes from what I know, that is why

Reply:
Okay just went through the code, I stand corrected by the fact that 1 game day is 74 ticks.
I would suggest to make it 5 game days, for the player to even acknowledge a problem, which in reality is only 10 seconds according to the timer_game_tick.h